### PR TITLE
METRON-547 Create integration components for Yarn and Zookeeper, use them in Maas Tests

### PR DESCRIPTION
**Pull request by ottobackwards**
Standardize the Maas Integration testing to use the Component/ComponentRunner system and create re-usable components for Yarn and Zookeeper

Test with mvn clean integration-test

### CODE REVIEW

File: metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/YarnComponent.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    }
+
+    public String getAppMasterJar(){
+        return appmasterJar;
+    }
+
+    public YarnConfiguration getConfig(){
+        return conf;
+    }
+
```
Comment (ottobackwards) [inline metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/YarnComponent.java]: One question here is to expose things like the node manager like this, as needed or to just expose the cluster itself.  Thoughts?

**Comment by cestella** [comment]
Could you please rename this preface METRON-547?

**Comment by ottobackwards** [comment]
sorry

File: metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/YarnComponent.java (hunk 1 above)
Comment (cestella) [inline metron-platform/metron-integration-test/src/main/java/org/apache/metron/integration/components/YarnComponent.java]: I'd just expose the `YarnCluster`, honestly.

**Comment by cestella** [comment]
I dig it.  +1 by inspection.

**Comment by ottobackwards** [comment]
changed based on review, thanks!

**Pull request closed without merging**